### PR TITLE
Remove closure-proxy leftovers

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q             | A
| ------------- | ---
| Branch?       | 3.3 
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no 
| Tests pass?   | yes
| Fixed tickets | N/A 
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php [hunk 1]
```diff
@@ -310,8 +310,6 @@ protected function sortServiceIds(array $serviceIds)
 
     protected function formatClosure(\Closure $closure)
```
Comment (ogizanagi) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php]: Should we revert entirely #22422 ? (minor BC break)

Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php]: The class is internal so we can do it

Comment (ogizanagi) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php]: Oh, indeed, I missed it. Reverted.

**Comment by fabpot** [comment]
Thank you @ogizanagi.

**Pull request merged**